Doc/library/calendar.rst

```diff
@@ -146,46 +146,61 @@ interpreted as prescribed by the ISO 8601 standard.  Year 0 is 1 BC, year -1 is
       the specified width, representing an empty day. The *weekday* parameter
       is unused.
 
-   .. method:: formatweek(theweek, w=0)
+   .. method:: formatweek(theweek, w=0, highlight_day=None)
 
       Return a single week in a string with no newline. If *w* is provided, it
       specifies the width of the date columns, which are centered. Depends
       on the first weekday as specified in the constructor or set by the
       :meth:`setfirstweekday` method.
 
+      .. versionchanged:: next
+         If *highlight_day* is given, this date is highlighted in color.
```

Comment:
Should this say “using terminal escape sequences”?

Reply:
Hmm, I think it's more useful to describe the end result rather than the "how".

Reply:
That was my intention: mention that you need to print the result to a terminal to see the highlight. If you don't, you'll see some garbage bytes.